packer/hetzner.json

```diff
@@ -0,0 +1,42 @@
+{
+  "variables": {
+    "location": "nbg1",
+    "api_token": "<YOUR_API_TOKEN>",
+    "xlt_version": "4.12.2",
+    "image_version": "4-12-2",
+    "label_xlt_version": "4-x"
```

Comment:
Do we want to use some more recent XLT version?

Reply:
Maybe this should be updated for all supported clouds at once. For now this would be at least consistent with the rest.

Reply:
Reasonable.